src/main/java/core/basesyntax/strategy/PurchaseHandler.java

```diff
@@ -0,0 +1,11 @@
+package core.basesyntax.strategy;
+
+import core.basesyntax.service.implementation.StorageServiceImpl;
+
+public class PurchaseHandler implements FruitShopOperationsHandler {
+    @Override
+    public void applyOperation(String product, int quantity) {
+        StorageServiceImpl storageService = new StorageServiceImpl();
```

Comment:
first of all - you have to use interface over implementation. Fix it in all places.
Is it a good idea to create a new class every time when you call the method? I think not. Put this parameter as a class level variable via the constructor. Fix in all places too

Reply:
done